src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionRoutes.scala

```diff
@@ -60,13 +61,15 @@ trait GoogleExtensionRoutes extends ExtensionRoutes with UserInfoDirectives with
             pathPrefix("key") {
               get {
                 complete {
-                  googleExtensions.getPetServiceAccountKey(WorkbenchUser(userInfo.userId, userInfo.userEmail), GoogleProject(project)).map(x => StatusCodes.OK -> x)
+                  import spray.json._
+                  // parse json to ensure it is json and tells akka http the right content-type
+                  googleKeyCache.getKey(WorkbenchUser(userInfo.userId, userInfo.userEmail), GoogleProject(project)).map(key => StatusCodes.OK -> key.parseJson)
```

Comment:
some of these changes duplicate effort in #81 

i'll rectify once that goes into dev